.gitignore

```diff
@@ -69,3 +69,6 @@ public/critical.css
 # Sphinx and MyST
 docs/_build/
 /.python-version
+
+# gitpod
+.gitpod.yml
```

Comment:
This is not your user's .gitignore. Please revert this change, and add it to `~/.gitignore_global`.